OpenRA.Mods.Common/Activities/Hunt.cs

```diff
@@ -44,4 +45,4 @@ public override Activity Tick(Actor self)
 				this);
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Not intentional, but desired. https://github.com/OpenRA/OpenRA/pull/10759#discussion_r52845580